apps/microtvm/zephyr/template_project/src/mlperftiny/main.cc

```diff
@@ -0,0 +1,37 @@
+/*
```

Comment:
I like how you combined the `submitter_implemented.cpp` files together here - this seems nice and clean.